lib/smart_answer_flows/marriage-abroad/outcomes/countries/mozambique/_title.govspeak.erb

```diff
@@ -0,0 +1,7 @@
+<% content_for :title do %>
+  <% if calculator.partner_is_same_sex? %>
+    Same-sex marriage and civil partnership in Mozambique
+  <% else %>
+      Marriage in Mozambique
```

Comment:
Good spot. I'll remove the spaces.  It doesn't seem to affect the test artefacts though, which is a relief.